Code/Framework/AzFramework/Tests/QualitySystemComponentTests.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/UnitTest/TestTypes.h>
+#include <AzCore/Console/Console.h>
+#include <AzCore/Settings/SettingsRegistry.h>
+#include <AzCore/Settings/SettingsRegistryImpl.h>
+#include <AzFramework/Quality/QualitySystemComponent.h>
+
+namespace UnitTest
+{
+    using namespace AZ;
+    using namespace AzFramework;
```

Comment:
Could you not do this?

We are trying to remove the number of using namespace at namespace scope, as this effects unity files and causes issues with unqualified names from OS include code conflicting with O3DE code.

For example we had a previous issue with `using namespace AZStd` in the UnitTest before
https://github.com/o3de/o3de/pull/7184